corehq/apps/export/templates/export/partials/table.html

```diff
@@ -120,6 +120,15 @@ <h4 ng-class="{'text-muted': export.emailedExport && !export.isAutoRebuildEnable
                                 {% trans "Just now" %}&nbsp;&nbsp;&nbsp;
                             </span>
                             <strong>{% trans "Last Downloaded:" %}</strong> {{ export.emailedExport.fileData.lastAccessed }}
+                            {% endangularjs %}
+                            {% if request|toggle_enabled:"SUPPORT" %}
+                                {% angularjs %}
+                                <span ng-show="export.last_runtime">
+                                   <strong>{% trans "Last Runtime:" %}</strong> {{ export.last_runtime }}
```

Comment:
This is cool

Reply:
`angular` works but `django` does not work or then vice-versa, had to add couple of `angularjs` / `endangularjs` , not fun.

Reply:
I'd call this `duration` rather than `runtime` to make it clearer what the value represents.

Reply:
thanks @snopoke . Updated.